### PR TITLE
replace ed25519 loaded from cdn

### PR DESCRIPTION
**Pull request by oliviathet**
ENG-1479

### CODE REVIEW

File: export/index.html [hunk 1]
```diff
@@ -774,7 +783,7 @@ <h2>Message log</h2>
         const privateKeyBytes = new Uint8Array(keyBytes);
         if (keyFormat === "SOLANA") {
           const privateKeyHex = TKHQ.uint8arrayToHexString(privateKeyBytes.subarray(0,32));
-          const publicKeyBytes = ed.getPublicKey(privateKeyHex);
+          const publicKeyBytes = TKHQ.getEd25519PublicKey(privateKeyHex);
```
Comment (andrewkmin) [inline export/index.html:786]: 👨‍🍳💋

**Review by andrewkmin — APPROVED** [review]
👏👏👏

🔥🔥🔥🚒🧨🧑‍🚒🧯

File: export/index.html [hunk 2]
```diff
@@ -565,10 +578,6 @@ <h2>Message log</h2>
   <script type="module">
     // TODO: this should be bundled at build time or replaced with code written by Turnkey entirely.
     import * as hpke from "https://esm.sh/@hpke/core";
```
Comment (andrewkmin) [inline export/index.html:580]: out of scope for this PR but I'd assume we'll want to tackle this as well at some point?

Comment (oliviathet) [reply export/index.html:580]: yep! this one's a big one so deserves a separate PR. also given that it's used by auth, recovery, export, and import, might want to reorganize the code a bit while we do it so there's less duplication

**Pull request merged**